frontend/src/utils/api.js

```diff
@@ -87,13 +86,20 @@ function publishAllBooksInYear(readingOlympicsYear) {
   };
   // return fetch(`${API_URL}/setYearOrSomething`, {
   //   method: 'POST',
+  // headers: {
+  //   'Content-Type': 'application/json'
+  // },
+  // credentials: 'include',
```

Comment:
Oh @Davidw1339 we need an endpoint to publish all books in a year..

Reply:
oh yup that should be through route /publish_books which takes in a year and boolean to publish or unpublish. https://github.com/hack4impact-uiuc/philareads/blob/master/backend/api/views/book.py#L251

Json request should be along the lines of
```JSON
{
    "year": 2019,
    "published": true
}
```
to publish all books in 2019